proto/internal/buf.yaml

```diff
@@ -12,6 +12,7 @@ breaking:
   # Uncomment this to temporarily ignore specific files or directories:
   ignore:
     - temporal/server/api/historyservice/v1/
+    - temporal/server/api/schedule/v1/message.proto
```

Comment:
Put a comment to remove this exception once this PR is merged?
Also we should remove the exception for historyservice.